src/components/BaseLayerSwitcher/BaseLayerSwitcher.js

```diff
@@ -143,41 +152,43 @@ function BaseLayerSwitcher({
       aria-label={altText}
       title={titles.closeSwitcher}
     >
-      <FaChevronCircleLeft size={15} focusable={false} />
+      <ChevronLeft />
     </div>
   );
 
   return (
     <div className={`${className}${openClass}`}>
       {!isClosed && toggleBtn}
       {!isClosed ? (
-        baseLayers.map((layer, index) => {
-          const layerName = layer.getName();
-          const activeClass =
-            layerName === currentLayer.getName() ? ' rs-active' : '';
-          return (
-            <div
-              key={layer.key}
-              className={`rs-base-layer-switcher-button ${openClass}${activeClass}`}
-              role="button"
-              title={t(layerName)}
-              aria-label={t(layerName)}
-              onClick={() => onLayerSelect(layer)}
-              onKeyPress={(e) => {
-                if (e.which === 13) {
-                  onLayerSelect(layer);
-                }
-              }}
-              style={getImageStyle(images[index])}
-              tabIndex="0"
-            >
-              <div className="rs-base-layer-switcher-title">{t(layerName)}</div>
-              {images[index] ? null : (
-                <span className="rs-alt-text">{t(altText)}</span>
-              )}
-            </div>
-          );
-        })
+        baseLayers
+          .sort((l1) => (l1.name === currentLayer.name ? -1 : 0))
```

Comment:
This does not seem to work.

Reply:
I tested it, it seems to work for me. But maybe it's not the best way to achieve the goal

Reply:
If this should move the current selected layer to the beginning of the selection list, it does not work for me. The general approach looks good to me.

Reply:
Since .sort() seems to behave slightly differently on different browsers I implemented a different sort method, see updates